src/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/Serialization/InstanceDescriptor.cs

```diff
@@ -150,10 +142,6 @@ public object Invoke()
             {
                 return ((FieldInfo)MemberInfo).GetValue(null);
             }
-            else
-            {
-                Debug.Fail($"Unrecognized reflection type in instance descriptor: {MemberInfo.GetType().Name}");
```

Comment:
You removed this presumably because you were able to trigger it with a test?

Reply:
Can be triggered with EventInfo